kat066/biglunchbox-pomodoro
# Ci cd update

==Changelog==
- Potentially fix resource not available errors
- Fixes no such file or directory errors
- Update cypress to v6.9.1

==Testing==
- Use nektos/act to locally run github actions
- Use https://github.com/AxiosDeminence/cse112-f22-group11 fork to run github actions in real environment

==Version Compatibility==
- N/A

==Detail Description==
- Resources not available error might not be fixed, but was not found as an issue on forked development.
- Any 'no such file or directory error' should be fixed by removing working-directory and prefixes unless absolutely necessary.
- Flatten repo to prevent headache with node_module issues.
- Cypress v6.5.0 was having issues with no longer being cached preventing it from being called. v6.9.1 is used in place.

==Potential Improvements==
- Unflatten the repo and move configuration files
- Make node installation use already available cache with --prefer-offline (requires testing if noteable speedup)
- Make Cypress use configuration matrix for cross-browser testing
- Create and upload documentation and coverage reports as artifacts in integration step for use in deploy step.
